### ISSUE TITLE
[Broken Website] learn.microsoft.com switches to "High Contrast" theme when opened in new tab

### ISSUE THREAD

**Issue by cal4** [darkreader/darkreader#11751]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

When Dark Reader is enabled and the website is opened in a new tab, the website overrides it and enables the high contrast theme (although website says light theme is selected). Happens whether Dark Reader's dark or light mode is enabled. Doesn't happen if Dark Reader is disabled for the site, unlike #3618.

### Website Address

https://learn.microsoft.com/en-us/

### Steps To Reproduce

1. Make sure Dark Reader is enabled for https://learn.microsoft.com/en-us/
2. Open https://learn.microsoft.com/en-us/ **in a new tab** by middle-click or right-click in background
3. Notice it high contrast theme is enabled (although website says light theme is selected)

### Screenshots

![image](https://github.com/darkreader/darkreader/assets/9409256/7b0f2de8-4e67-4dc4-b432-8f8ba0b8acb2)
![image](https://github.com/darkreader/darkreader/assets/9409256/3261b399-8d60-49b9-b698-865de30a5743)

### Operating System

Windows 11 version 21H2 (0S Build 22000.2295), Windows 10 Version 22H2 (OS Build 19045.3448

### Web Browser name and version

Google Chrome Version 117.0.5938.92 (Official Build) (64-bit)

### Dark Reader version

Version 5 Preview (4.9.65)

### Additional Context

Works normally if link is opened in current tab.

**Comment by SAMURAii-7** [comment]
Not reproducible even after spamming new tabs. Does not seem to be a problem due to Dark Reader
![image](https://github.com/darkreader/darkreader/assets/61153966/e32b8368-a582-47a8-97ac-a3680ed39ae0)

**Comment by SAMURAii-7** [comment]
Nope, but I don't think that should make a difference. Maybe other contributors can confirm whether it's reproducible or not. Or if you want you can try fixing it yourself, refer to CONTRIBUTING.md

**Comment by cal4** [comment]
To reproduce, you have to open the website in a background tab. Otherwise, it works correctly.

**Comment by SAMURAii-7** [comment]
Ok I will try that. In your initial comment on how to reproduce the issue, it said to "open a new tab"
![image](https://github.com/darkreader/darkreader/assets/61153966/89d2b599-6af6-4b25-9160-c2e7614dd3e0)
Please make sure the information is correct before raising an issue in the future

**Comment by SAMURAii-7** [comment]
Issue is reproducible, will try to work on a fix

**Comment by cal4** [comment]
Thanks for taking a look!

**Comment by SAMURAii-7** [comment]
I am not familiar enough with the codebase to resolve this issue 😅 maybe @Gusted can take a look because it seems the issue might be related to #3618. I will be ready to help in any way I can

**Comment by cal4** [comment]
Worth noting that this bug occurs on https://docs.microsoft.com/en-us/ as well, like #3618, except excluding the page works properly.

**Comment by Merlin2001** [comment]
Also on https://learn.microsoft.com/ and for me, the exclusion _doesn't_ work.  
I have Dark Reader set up to "Invert listed only" and the Microsoft sites are not listed. As mentioned in https://github.com/darkreader/darkreader/issues/3618#issuecomment-894857887 my Windows UI is set to Dark Mode and the second refresh of a Microsoft Learning or Docs page makes it go into high contrast mode :(
